### DISCUSSION TITLE
Real World Data Science - Creating Christmas cards with R

### DISCUSSION THREAD

**Discussion by giscus** [realworlddatascience/realworlddatascience.github.io#142] [General]
# Real World Data Science - Creating Christmas cards with R

The programming language R is capable of creating a wide variety of geometric shapes that can be used to construct high quality graphics – including festive images. In this tutorial, Nicola Rennie walks us through the process of using R packages to create and send Christmas cards with R.

https://realworlddatascience.net/ideas/tutorials/posts/2023/12/12/xmas-cards.html?utm_campaign=Data_Elixir&utm_source=Data_Elixir_464

**Comment by myrtifolia** [comment]
Thank you for sharing. It was fun!

I have one question though. When I save it as png, it produces black lines (square, basically) on the image file. Is there a way to avoid it?

**Reply by brtarran** [reply]
If the black lines run around the edge of the image as a border, I had this same issue. Not sure if my solution is the most elegant, but I added a `color` element to `plot.background` so that the border `color` matches the background `fill` colour:

```
s1 <- ggplot() +
  theme_void() +
  theme(
    plot.background = element_rect(fill = "#939bc9", color = "#939bc9")
  )
s1
```

https://realworlddatascience.net/viewpoints/editors-blog/posts/2023/12/12/rwds-xmas-card.html

**Reply by myrtifolia** [reply]
Elegant enough :). 

Thank you. Now everything looks like it should be.
